docs/GraFx-Developers/grafx-studio/enduser/index.md

```diff
@@ -0,0 +1,42 @@
+# Integrate GraFx Studio Enduser UI
+
+## Introduction
+
+**Making Smart Templates**
+
+To make Smart Templates in GraFx Studio, you'll need a "template designer seat." Once your template is ready, it moves to the stage where end users can interact with it, specifically in "[My Projects](/GraFx-Studio/guides/create-projects/)." This area is where templates come to life, allowing end users to consume and customize them to fit their needs.
+
+**In your webportal**
+
+Moreover, this end-user interface isn't limited to just the CHILI GraFx environment; it's designed for broader application and can be seamlessly integrated into custom web portals. 
+
+This flexibility opens up numerous possibilities for customizing and enhancing the user experience. For detailed instructions on integrating the end-user interface into your web portal, refer to the guide below.
+
+## Token
+
+!!! warning
+	Before you start, take note on how to generate a token.
+
+Since you don't want to pass secure credentials, your server side code will need to generate a token.
+
+[This page](/CHILI-GraFx/guides/integrations/#step-4-generating-an-access-token) explains how.
+
+## Quick start
+
+On this page on Github, you can get started in no time to have your own GraFx Studio Enduser UI integrated.
```

Comment:
I think this could be a bit clearer, maybe a link to the github page tied to the "this page on Github" text? The link is already at the bottom of the section but it's kind of awkward to reference it here without a link immediately available